modules/gapi/test/cpu/gapi_ocv_stateful_kernel_tests.cpp

```diff
@@ -387,6 +530,45 @@ TEST(StatefulKernel, StateIsChangedViaCompArgsOnReshape)
     run("cv/video/768x576.avi", "knn");
     run("cv/video/1920x1080.avi", "mog2");
 }
+
+TEST(StatefulKernel, StateIsResetOnceOnReshapeInStreaming)
```

Comment:
Why do we care about reshape in streaming?
If source changed, reset should happen - isn't it?

Reply:
This test is here to highlight an issue that we can have 2 setup() calls in such cases